RubeesEat/Pages/UserHomepage.cshtml

```diff
@@ -0,0 +1,21 @@
+@page
+@inject IBillRepository billRepository
+@inject IPersonRepository personRepository
+
+Hello, @personRepository.GetCurrentUser().FirstName!
+
+
+<ul id="balanceChanges" style="list-style: none">
+    @foreach (var item in billRepository.GetRecentBalanceChanges(personRepository.GetCurrentUser().Id, 10))
+    {
+        @*@item.Amount < 0*@
+        <li>
+
+            <div style="display: inline; color: @(item.Amount < 0 ? "red" : "green");">@item.Amount€</div>
+
+            @item.Date.DayOfWeek
+            @item.Date.ToString("dd/MM/yyyy")
```

Comment:
I would suggest using a format specifier that already includes the the day of the week like `"D"` (see https://learn.microsoft.com/en-us/dotnet/standard/base-types/standard-date-and-time-format-strings). This way, we only have one statement to format the date and it uses the current culture to format the date.